spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplateTests.java

```diff
@@ -148,6 +159,22 @@ public void readOnlyTransactionTest() {
 		verify(readOnlyTransaction, times(2)).read(eq("custom_test_table"), any(), any());
 	}
 
+	@Test(expected = SpannerDataException.class)
```

Comment:
please check exception messages, not just exception classes

Reply:
I think this is gonna need to be a separate issue. We have about 600 tests in repo and about 280 of them are in Spanner, and only recently have we started checking the messages in Datastore.

#1225